samples/Azure.Management.Storage/Generated/RestOperations/BlobServicesRestOperations.cs

```diff
@@ -271,6 +271,8 @@ public async Task<Response<BlobServiceData>> GetServicePropertiesAsync(string re
                         value = BlobServiceData.DeserializeBlobServiceData(document.RootElement);
                         return Response.FromValue(value, message.Response);
                     }
+                case 404:
```

Comment:
I don't think this one should get the 404

Reply:
Updated to remove for singleton case as it does not have a main get method in operations.